beetsplug/fetchart.py

```diff
@@ -288,26 +288,45 @@ def fetch_image(self, candidate, plugin):
 
 
 class CoverArtArchive(RemoteArtSource):
-    NAME = u"Cover Art Archive"
-
     if util.SNI_SUPPORTED:
         URL = 'https://coverartarchive.org/release/{mbid}/front'
         GROUP_URL = 'https://coverartarchive.org/release-group/{mbid}/front'
     else:
         URL = 'http://coverartarchive.org/release/{mbid}/front'
         GROUP_URL = 'http://coverartarchive.org/release-group/{mbid}/front'
 
+    def getReleaseCandidate(self, album):
```

Comment:
This is a tiny thing, but our (and [PEP 8](https://www.python.org/dev/peps/pep-0008/)'s) naming style for methods uses snake_case: i.e., `get_release_candidate`.